src/Sylius/Behat/Context/Api/Shop/CustomerContext.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Shop;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Behat\Context\Setup\ShopSecurityContext;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Webmozart\Assert\Assert;
+
+final class CustomerContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $accountClient;
```

Comment:
Should be changed I guess
```suggestion
    private $customerClient;
```